components/pages/project/version/_version/index.vue

```diff
@@ -297,7 +297,11 @@ export default {
 
       await this.$axios.delete(`version/${this.version.id}`, this.$auth.headers)
 
-      await this.$router.replace(`/mod/${this.mod.id}`)
+      await this.$router.replace(
+        `/${this.project.project_type}/${
+          this.project.slug ? this.project.slug : this.project.id
+        }`
+      )
```

Comment:
https://knossos-venashial-qsky5l87i-venashial.vercel.app/mod/fred/version/nTPy9wmt
^ This works, whereas
https://knossos-venashial-qsky5l87i-venashial.vercel.app/mod/fred/version/v1.fred
gives `500: INTERNAL_SERVER_ERROR Code: FUNCTION_INVOCATION_FAILED`.

Reply:
Not sure what caused the error, but it appears to be fixed in the latest deploy